lib/pages/chat_list.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:lend_a_hand/pages/conversation_page.dart';
+
+class ChatList extends StatefulWidget {
+  final FirebaseUser firebaseUser;
+
+  const ChatList({Key key, this.firebaseUser}) : super(key: key);
+
+  @override
+  _ChatListState createState() => _ChatListState();
+}
+
+class _ChatListState extends State<ChatList> {
+  final Firestore _firestore = Firestore.instance;
+
+  var _conversations;
+
+  @override
+  void initState() {
+    super.initState();
+
+    // TODO: Need to create link between chat documents within collection to user's active chats to build out conversations list, currently mocked.
+    _conversations = FutureBuilder<QuerySnapshot>(
+      future: _firestore.collection("chats")
+          .getDocuments(),
+      builder: (context, snapshot) {
+        if (!snapshot.hasData)
+          return Center(child: CircularProgressIndicator());
+
+        print(snapshot.hasData);
+        print(snapshot.data.documents.isEmpty);
+
+        List<Widget> conversations = snapshot.data.documents.map((doc) =>
+            new UserConversation(email: doc.documentID, firebaseUser: widget.firebaseUser)
+        ).toList();
+
+        return ListView(
+          children: <Widget>[...conversations],
+        );
+      },
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+       appBar: AppBar(
```

Comment:
We probably will need to remove AppBar, I think it will show 2 times if we put this as a page in the navigation tab bar.